### PR TITLE
LPS-163223 Fix poshi automation | master

### PR DESCRIPTION
**Pull request by ozysouza**
https://issues.liferay.com/browse/LPS-163223

cc @david-gutierrez-mesa 
cc @Tim-Cao

### CODE REVIEW

**Comment by liferay-echo** [comment]
Thank you for submitting this PR. 👏

Make sure it follows the [Echo Pull Request Template](https://gist.github.com/liferay-echo/5a4455285ff1800f9da146816d85e67d#file-pull-request-template-md) if you haven't already done so. 

Learn more about it [here](https://liferay.atlassian.net/wiki/spaces/ENGECHO/pages/2027455014/Echo+Pull+Request+Etiquette).

File: portal-web/test/functional/com/liferay/portalweb/tests/enduser/wem/layout/contentpage/ContentPagesWithStaging.testcase [hunk 1]
```diff
@@ -1685,89 +1685,93 @@ definition {
...
+		task ("When the page creator previews the draft page via header ellipsis icon of page editor") {
 			ContentPages.gotoPageEditor();
 
 			PageEditor.previewInANewTab();
-
-			ContentPages.viewCurrentExperience(experienceName = "New Experience");
 		}
 
-		task ("Back to view page mode and view the new experience still selected in experience selector") {
-			Close.closeWindow();
```
Comment (Tim-Cao) [inline portal-web/test/functional/com/liferay/portalweb/tests/enduser/wem/layout/contentpage/ContentPagesWithStaging.testcase:1766]: Shouldn't remove

File: portal-web/test/functional/com/liferay/portalweb/tests/enduser/wem/layout/contentpage/ContentPagesWithStaging.testcase [hunk 2]
```diff
@@ -1685,89 +1685,93 @@ definition {
...
-
-			PageEditor.clickBack();
```
Comment (Tim-Cao) [inline portal-web/test/functional/com/liferay/portalweb/tests/enduser/wem/layout/contentpage/ContentPagesWithStaging.testcase:1768]: Shouldn't remove

File: portal-web/test/functional/com/liferay/portalweb/tests/enduser/wem/layout/contentpage/ContentPagesWithStaging.testcase [hunk 3]
```diff
@@ -1685,89 +1685,93 @@ definition {
...
+		task ("The the page creator should see the fragments on New Experience") {
+			ContentPages.viewFragmentText(
+				fragmentName = "button",
+				id = "link",
+				text = "Go Somewhere");
 
-			ContentPages.viewCurrentExperience(experienceName = "New Experience");
```
Comment (Tim-Cao) [inline portal-web/test/functional/com/liferay/portalweb/tests/enduser/wem/layout/contentpage/ContentPagesWithStaging.testcase:1770]: Keep the origin

**Comment by Tim-Cao** [comment]
See changes @ozysouza
https://github.com/liferay-echo/liferay-portal/pull/9645/commits/d8bdfc1c024b2a28921d2a4e330a250c7ec14483

**Comment by Tim-Cao** [comment]
ci:close

**Pull request closed without merging**